cmd/ci-operator/main.go

```diff
@@ -786,6 +788,29 @@ func overrideMultiStageParams(o *options) error {
 	return nil
 }
 
+// applyEnvOverrides processes environment variables with override prefixes and applies them to the test configurations.
```

Comment:
Comment is nice but a bit outdated:
```golang
It checks for environment variables that start with "OVERRIDE_"
```
not the case anymore, it's `MULTISTAGE_PARAM_OVERRIDE_`